crates/client/mapping-sync/src/block_metrics.rs

```diff
@@ -6,8 +6,10 @@ pub struct BlockMetrics {
     pub block_height: Gauge,
     pub transaction_count: Counter,
     pub event_count: Counter,
-    pub l1_gas_price_wei: Gauge,
-    pub l1_gas_price_strk: Gauge,
+    pub eth_l1_gas_price_wei: Gauge,
+    pub strk_l1_gas_price_fri: Gauge,
+    pub eth_l1_data_gas_price_wei: Gauge,
```

Comment:
maybe blob is a better prefix than data

Reply:
I don't chose the names, they are set by starknet_api/blockifier, better be consistent